index.html

```diff
@@ -0,0 +1,292 @@
+      <div class="program-container">
+        <div class="main-text">Main Program</div>
+        <div class="orange-line"></div>
+      </div>
+      <div class="program-details-container">
+        <div class="programe-detail">
+          <img
+            src="./images/phone-view-icons/program_icon_01.png"
+            alt="conference-icon1"
+          />
+          <span class="title-programe">Discours</span>
+          <p class="programe-para">
+            Listen to the opening talks of the manager about the restaurant.
+          </p>
+        </div>
+        <div class="programe-detail">
+          <img
+            src="./images/phone-view-icons/program_icon_02.png"
+            alt="conference-icon2"
+          />
+          <span class="title-programe">Presentation</span>
+          <p class="programe-para">
+            Video presentented on a large screen of diffrerent types of food
+            present in our various menu.
+          </p>
+        </div>
+        <div class="programe-detail">
+          <img
+            src="./images/phone-view-icons/program_icon_03.png"
+            alt="conference-icon3"
+          />
+          <span class="title-programe">Forum</span>
+          <p class="programe-para">
+            Listen to diffrerent ways of improving our service to have a better
+            experience.
+          </p>
+        </div>
+        <div class="programe-detail">
+          <img
+            src="./images/phone-view-icons/program_icon_04.png"
+            alt="conference-icon4"
+          />
+          <span class="title-programe">Exchange</span>
+          <p class="programe-para">
+            Opportunity open to recrute good personnel.
+          </p>
+        </div>
+        <div class="programe-detail">
+          <img
+            src="./images/phone-view-icons/program_icon_05.png"
+            alt="conference-icon5"
+          />
+          <span class="title-programe">Degustation</span>
+          <p class="programe-para">
+            party time and enjoyement will be available.
+          </p>
+        </div>
+      </div>
+      <div class="btn-mobile-program">SEE THE WHOLE PROGRAM</div>
+      <button type="button" class="btn-program">Join Us</button>
+    </section>
+    <section class="investors">
+      <h3>Featured Investors</h3>
+      <div class="orange-line"></div>
+      <div class="investors-container">
+        <div class="investors-list">
+          <img
+            src="./images/phone-view-icons/investors_01.png"
+            alt="investors-icon1"
+            class="investors-img"
+          />
+          <div class="investors-content">
+            <h4>Yochai Benkler</h4>
+            <h5>Chef in Hotel-hilton</h5>
+            <hr class="investors-line" />
+            <p class="investors-description">
+              Focusing on a common approach in how to improve the quality of the
+              service in a restaurant.
+            </p>
+          </div>
+        </div>
```

Comment:
- Kindly, center the investor list and adjust the space between them to stick to the given design and have better UI 👍 
#### your Design 👇
![Screenshot 2023-07-06 141317](https://github.com/axcell-0/first-capstone/assets/52010789/e56a5191-86a6-4774-8d4b-c6eae056f6ff)

#### Given Design 👇
![Screenshot 2023-07-06 141154](https://github.com/axcell-0/first-capstone/assets/52010789/c1c79ba5-be6d-4c8a-93c9-f3cb3e95d462)